backend-api/routes/api.php

```diff
@@ -12,7 +11,29 @@
 | is assigned the "api" middleware group. Enjoy building your API!
 |
 */
-
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
```

Comment:
`/user` is the only route that needs authentication?

Reply:
Question: we also have declared the middleware inside AuthController. Should we move all the middleware declarations to api.php?
![image](https://user-images.githubusercontent.com/56364209/72178449-2d5ec600-33e3-11ea-80a2-2b4fc022e5ac.png)